ocaml/idl/datamodel_types.ml

```diff
@@ -58,38 +58,179 @@ let rel_dundee_plus = "dundee-plus"
+      branding      = "XCP 0.5";
+      xs_branding   = "Unreleased";
+    }; {
+      code_name     = None;
+      version_major = 1;
+      version_minor = 5;
+      branding      = "XCP 0.5";
+      xs_branding   = "XenServer 5.0 update 3";
+    }; {
+      code_name     = Some rel_george;
+      version_major = 1;
+      version_minor = 6;
+      branding      = "XCP 0.5";
+      xs_branding   = "XenServer 5.5";
+    }; {
+      code_name     = Some rel_midnight_ride;
+      version_major = 1;
+      version_minor = 7;
+      branding      = "XCP 0.5";
+      xs_branding   = "XenServer 5.6";
+    }; {
+      code_name     = Some rel_cowley;
+      version_major = 1;
+      version_minor = 8;
+      branding      = "XCP 1.0";
+      xs_branding   = "XenServer 5.6 FP1";
+    }; {
+      code_name     = Some rel_boston;
+      version_major = 1;
+      version_minor = 9;
+      branding      = "XCP 1.5";
+      xs_branding   = "XenServer 6.0";
+    }; {
+      code_name     = Some rel_tampa;
+      version_major = 1;
+      version_minor = 10;
+      branding      = "XCP 1.6";
+      xs_branding   = "XenServer 6.1";
+    }; {
+      code_name     = Some rel_clearwater;
+      version_major = 2;
+      version_minor = 0;
+      branding      = "XenServer 6.2";
+      xs_branding   = "XenServer 6.2";
+    }; {
+      code_name     = Some rel_vgpu_tech_preview;
+      version_major = 2;
+      version_minor = 0;
+      branding      = "XenServer 6.2 vGPU preview";
+      xs_branding   = "XenServer 6.2 SP1 Tech-Preview";
+    }; {
+      code_name     = Some rel_vgpu_productisation;
+      version_major = 2;
+      version_minor = 1;
+      branding      = "XenServer 6.2 SP1";
+      xs_branding   = "XenServer 6.2 SP1";
+    }; {
+      code_name     = Some rel_clearwater_felton;
+      version_major = 2;
+      version_minor = 2;
+      branding      = "XenServer 6.2 SP1 Hotfix 4";
+      xs_branding   = "XenServer 6.2 SP1 Hotfix 4";
+    }; {
+      code_name     = Some rel_clearwater_whetstone;
+      version_major = 2;
+      version_minor = 2;
+      branding      = "XenServer 6.2 SP1 Hotfix 11";
+      xs_branding   = "XenServer 6.2 SP1 Hotfix 11";
+    }; {
+      code_name     = Some rel_creedence;
+      version_major = 2;
+      version_minor = 3;
+      branding      = "XenServer 6.5";
+      xs_branding   = "XenServer 6.5";
+    }; {
+      code_name     = Some rel_cream;
+      version_major = 2;
+      version_minor = 4;
+      branding      = "XenServer 6.5 SP1";
+      xs_branding   = "XenServer 6.5 SP1";
+    }; {
+      code_name     = Some rel_indigo;
+      version_major = 2;
+      version_minor = 4;
+      branding      = "indigo (unreleased)";
+      xs_branding   = "Unreleased";
+    }; {
+      code_name     = Some rel_dundee;
+      version_major = 2;
+      version_minor = 5;
+      branding      = "XenServer 7.0";
+      xs_branding   = "XenServer 7.0";
+    }; {
+      code_name     = Some rel_dundee_plus;
```

Comment:
Is is used in the datamodel, for example PVS is under dundee-plus, which means it's too late to change it :(